mautic/mautic
# [UX] Tokens help

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR adds an entire modal to show general tokens.
Tokens for created custom fields are not included.

![image](https://github.com/mautic/mautic/assets/116097999/6f35ed80-f9a1-4458-8b96-9bf488728c82)

![image](https://github.com/user-attachments/assets/3c5ee1eb-d60e-4779-a8cb-b15e823b0648)








---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. At the top navbar, click on the help icon
3. Click on placeholder tokens
4. Interact
